codegen/smithy-dafny-codegen/src/test/java/software/amazon/polymorph/traits/ValidatorTest.java

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.amazon.polymorph.traits;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.commonmark.node.Node;
+import org.commonmark.parser.Parser;
+import org.junit.Test;
+
+public class ValidatorTest {
```

Comment:
Nit: Weird name for this class, but I am not blocking on it.

Reply:
Fair, I was thinking we'll likely want more validator unit tests in the future. Will improve at next opportunity.